### ISSUE TITLE
Exploring the Effectiveness of Huber Loss Over MSE in Time-MOE Models: Insights from Ablation Studies"

### ISSUE THREAD

**Issue by iganggang** [Time-MoE/Time-MoE#24]
Hello author, I printed your paper. In the ablation study, the MSE of time-moe is reduced by 0.005 with Huber loss. In the loss function introduction of the paper, you said that the Huber loss function is used for large-scale parameters. I want to know if this function can be better than the ordinary MSE in a small model, such as DLinear or PatchTST. Could you share the code?

**Comment by Maple728** [comment]
Thank you for your interest in our work!

The Huber loss is particularly suitable for large-scale general datasets that contain numerous outlier values after zero standard normalization. This is because the Huber loss is robust to outliers, allowing for more stable model training.

The Huber loss is implemented in PyTorch, and you can use it directly as follows:

```python
loss_function = torch.nn.HuberLoss(reduction='none', delta=2.0)
```

Please feel free to contact us if you have any questions.

**Comment by iganggang** [comment]
I test it. it is also useful. MSE decreased 0.5%. Thanks a lot.

**Comment by Maple728** [comment]
I am grateful to hear it is useful to you.
